src/Oscar.jl

```diff
@@ -233,6 +233,23 @@ function open_doc()
     end
 end
 
+
+@doc Markdown.doc"""
+    build_doc(; doctest=false, strict=false)
+
+Build the documentation of `Oscar.jl` locally and open the front page in a
+browser.
+
+The optional parameter `doctest` can take three values:
+  - `false`: Do not run the doctests (default).
+  - `true`: Run the doctests and report errors.
+  - `:fix`: Run the doctests and replace the output in the documentation with
+    the output produced by Oscar. Please use this option carefully.
```

Comment:
I added a warning at the beginning, since I am unsure whether we can everyone to juggle different Julia versions on their machine without getting confused or confusing vs code. Then I thought that some people tend to not read the warnings at the beginning, so I added it to the end as well...